spark_expectations/notifications/plugins/email.py

```diff
@@ -99,14 +143,7 @@ def send_notification(
                 msg["To"] = _context.get_to_mail
                 msg["Subject"] = _context.get_mail_subject
 
-                # body = _config_args.get('mail_body')
-                mail_content = f"""{_config_args.get("message")}"""
-
-                # Check if the content is HTML
-                if re.search(r"<[a-z][\s\S]*>", mail_content, re.IGNORECASE):
-                    content_type = "html"
-                else:
-                    content_type = "plain"
+                mail_content, content_type = self._process_message(_context, _config_args)
```

Comment:
👍 

More as a reminder but not must or needed as part of your current story... As you did with processing message body we should probably extract into separate methods for readability and testing
- building `msg= MIMEMultipart` object and returning `text = msg.as_string()`
- actual mail sending